docs/integrations/sources/amazon-seller-partner.md

```diff
@@ -104,6 +104,7 @@ Make sure to configure the [required parameters](https://developer-docs.amazon.c
 
 | Version  | Date       | Pull Request                                               | Subject                                                                |
 |:---------|:-----------|:-----------------------------------------------------------|:-----------------------------------------------------------------------|
+| `0.2.28` | 2022-10-20 | [\#18283](https://github.com/airbytehq/airbyte/pull/18283) | Added multiple (21) report types                                       |
```

Comment:
Add newly added streams to the section `Supported streams`.

Reply:
done